bundle-workflow/python/build.py

```diff
@@ -3,31 +3,57 @@
 
 import os
 import sys
+import subprocess
 import tempfile
-import urllib.request
-from lib.build_manifest import BuildManifest
-from lib.build_output import BuildOutput
+import yaml
+from manifests.input_manifest import InputManifest
+from build_workflow.build_recorder import BuildRecorder
+from build_workflow.builder import Builder
+from build_workflow.git_repository import GitRepository
 
 if (len(sys.argv) < 2):
     print("Build an OpenSearch Bundle")
     print("usage: build.sh /path/to/manifest")
     exit(1)
 
-with tempfile.TemporaryDirectory() as work_dir:
-    manifest = BuildManifest.from_file(sys.argv[1])
-    build = manifest.build
-    output = BuildOutput()
+component_scripts_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/bundle-build/components')
+default_build_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/bundle-build/standard-gradle-build/build.sh')
+
+def get_arch():
```

Comment:
we are going to need to reuse this during assembly, can we extract this somewhere with static access?

Reply:
This already exists in https://github.com/opensearch-project/opensearch-build/blob/2ac8a7a876f497d7870fc591dbf497f0606cd018/bundle-workflow/python/lib/build_output.py#L25

Reply:
Oh, man, that is soooo the wrong place to figure out the architecture!

Reply:
> we are going to need to reuse this during assembly, can we extract this somewhere with static access?

Assembly will just read the architecture string from the build manifest, won't it?

Reply:
I was thinking I would need to validate it during bundling but that process is architecture independent.